src/test/com/abstract_calss/AnimalService.java

```diff
@@ -0,0 +1,41 @@
+package test.com.abstract_calss;
+
+/**
+ * Потрібно створити абстрактний клас Animal
+ * і його наслідники Hawk, Horse.
+ * У класі Animal будуть 2 абстрактних * методи move та talk,
+ * потрібно імплементувати в кожному з класів (Hawk та Horse) ці методи по різному, в одному
+ * буде fly, в іншому run, це треба вивести в консоль.
+ * Також потрібно зробити клас AnimalService з методом callAnimal,
+ * котрий приймає Animal і всередині викликає методи talk та move,
+ * для кожного з об’єктів і далі все це вивести у методі  main класу AnimalService.
+ * <p>
+ * Кожне завдання треба щось перевірити, для цього потрібен вивід у консоль як стандарт.
+ * В даному завданні ти створив абстрактний клас та 2 його наслідники,
+ * але щоб вивести його у консоль з демонстрацією того навіщо потрібні абстракції  нам треба AnimalService,
+ * там ти будеш в методі callAnimal приймати Animal animal і в залежності від того якого наслідника
+ * будеш туди передавати будеш отримувати різні повідомлення в консолі.
+ * <p>
+ * Потім у методі main ти створиш коня і орла і також обʼєкт сервіса і будеш у сервіс
+ * передавати спочатку коня потім орла і вони будуть по різному рухатися.
+ * Це дозволить наглядно показати що робить абстрактний метод
+ * Ну так у тому ж прикол абстрактного класу, що ти робиш абстракцію,
+ * потім декілька імплементацій в яких заставляєш користувача імплементувати абстрактні методи.
+ * Також після виконання завдання, ти можеш створити клас Human з реалізацією move типу walk і передати метод сервіса і він буде працювати.
+ */
+public class AnimalService {
+    public static void callAnimal(Animal animal) {
```

Comment:
empty line. 
We don't use static methods by default, by default we don't use them.